app/views/help/modifiying_requests.md

```diff
@@ -1,28 +1,27 @@
-<% title "What if there's something wrong with the request?" %>
+<% title "What if I need to modify a request?" %>
 
-A request may need to be modified after it is submitted. These situations include:
-
-* There was a typo.
-* A different RWA number needs to be used. (BA80)
-* The original request had a Not-To-Exceed amount, which needs to be changed to Exact once it's complete.
+You may need to modify a request after it is submitted for a number of reasons:
+  - To correct a typo or other error
+  - To update a Not-To-Exceed amount to a final purchase price
+  - To change approval information or provide a new RWA number
 
 ## Approver
 
-If you are one of the approvers and notice an issue:
+Approvers cannot modify requests, but they can ask the requester to do so. If you are one of the approvers and notice an issue:
```

Comment:
[Not actually true](https://github.com/18F/C2/blob/d416f246b6cd1c14bcd33439e81c9097369858ae/app/policies/ncr/work_order_policy.rb#L10-L12) for NCR. Wonder if we should make everyone involved able to edit by default anyway.